pkg/web/transactions.go

```diff
@@ -255,23 +335,47 @@ func (s *Server) UpdateTransaction(c *gin.Context) {
 	c.Negotiate(http.StatusOK, gin.Negotiate{
 		Offered:  []string{binding.MIMEJSON, binding.MIMEHTML},
 		Data:     out,
-		HTMLName: "transaction_update.html",
+		HTMLName: "transaction_info.html",
 	})
 }
 
-func (s *Server) DeleteTransaction(c *gin.Context) {
+func (s *Server) SendEnvelopeForTransaction(c *gin.Context) {
```

Comment:
This method is the primary function for review in this file. The rest of the changes are just re-organization and moving things around.